roles/matrix-nginx-proxy/defaults/main.yml

```diff
@@ -6,6 +6,9 @@ matrix_nginx_proxy_enabled: true
 matrix_nginx_proxy_docker_image: "nginx:1.19.0-alpine"
 matrix_nginx_proxy_docker_image_force_pull: "{{ matrix_nginx_proxy_docker_image.endswith(':latest') }}"
 
+matrix_nginx_proxy_proxy_synapse_worker_processes: 1
```

Comment:
Looks great! And thank you for this contribution! :+1: 

Why put `_synapse_` in the variable name when it affects all matrix-nginx-proxy vhosts?

Perhaps we should rename these variables to `matrix_nginx_proxy_worker_processes` and `matrix_nginx_synapse_worker_connections`.

Reply:
Yes you are right. I will rename the variables later and provide contribution again.
Thank you and all other contributors for this great project.